src/electionguard/encrypt.cpp

```diff
@@ -282,11 +284,11 @@ namespace electionguard
 
         // this method runs off to look in the precomputed values buffer and if
         // it finds what it needs then the returned class will contain those values
-        precomputedTwoTriplesAndAQuad = PrecomputeBufferContext::getTwoTriplesAndAQuadruple();
+        precomputedTwoTriplesAndAQuad = PrecomputeBufferContext::popTwoTriplesAndAQuadruple();
```

Comment:
the name here was changed to adapt to the new semantics. `popTwoTriplesAndAQuadruple` will pop off the queue while `getTwoTriplesAndAQuadruple` will try to pop but then create one if it fails. i need to do a bit more testing around this to see if i can simplify the control flow in that way